### DISCUSSION TITLE
Welcome to s2_c_programs Discussions!

### DISCUSSION THREAD

**Discussion by ashermatsh** [Asher-MS/s2_c_programs#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ashermatsh** [comment]
<h1>Share Your Ideas and Programs</h1>

**Comment by adamsyy** [comment]
🤔

**Reply by ashermatsh** [reply]
☺☺
